backend/validation/format.js

```diff
@@ -275,6 +288,36 @@ var verifyCourseCode = async courseCode => {
   }
 };
 
+/**
+ * Verifies that the title is not null
+ * @param {String} title
+ */
+var verifyTitle = async title => {
+  if (!title){
+		throw new Error("invalid format title");
+	}
+};
+
+/**
+ * Verifies that the department sub name is in the following format: XXXX
+ * @param {String} department
+ */
+var verifyDepartmentSubName = async department => {
+	if (!department || (String(department).length !== MAX_DEPARTMENTSUBNAME_LENGTH) || !isAlphabet(department)){
+		throw new Error("invalid format department name");
+	}
+};
+
+/**
+ * Verifies that the phased_out is either 0 or 1
+ * @param {String} phaseOut
+ */
+var verifyPhaseOut = async phaseOut => {
+	if (!phaseOut || !(String(phaseOut).match(/^[01]$/i)) ){
```

Comment:
Not sure what the formatting is checking here, phased out should be a boolean

Reply:
He assumes that either 0 (false) or 1 (true) is transmitted from front end

Reply:
does casting a boolean to a string return a digit (0/1) or "false"/"true" if its the later then this wont work